### PR TITLE
Remove discord log channel

### PR DESCRIPTION
**Pull request by jendrsczok**
RemoveDiscordLogChannel command added

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1428,6 +1401,12 @@
       "integrity": "sha512-kssjab8CvdXfcXMXVcvsXum4Hwdq9XGtRD3TteMEvEbq0LXyiNQr6AprqKqfeaDXze7SxWvRxdpwE6ku7ikLkg==",
       "dev": true
     },
+    "html-escaper": {
```
Comment (ogrady) [inline package-lock.json:1404]: what is this dependency for?

**Review by ogrady — CHANGES_REQUESTED** [review]
Looks mostly fine to me 🙂 but please remove the backup files (.orig). Also, Windows seems to have messed up the file permissions of _all_ files. Kindly revert them back to non-executable. This is apparently a known problem with windows: https://stackoverflow.com/questions/6476513/git-file-permissions-on-windows

File: package-lock.json (hunk 1 above)
Comment (jendrsczok) [reply package-lock.json:1404]: It comes from the update of the "istanbul-reports" (I don't know what that stands for).   The requirements have been changed from "handlebars": "^4.1.2" to "html-escaper" changed: "^2.0.0".

**Comment by jendrsczok** [comment]
It's done. Deleted two *.orig files and rollback the permissions.

**Pull request merged**